sdk/formrecognizer/ai-form-recognizer/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,125 @@
+# Troubleshoot Azure Form Recognizer client library issues
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Form Recognizer client library for Javascript.
+
+## Table of Contents
+
+- [Troubleshooting Errors](#troubleshooting-errors)
+  - [Handling RestError](#handling-resterror)
+  - [Build model error](#build-model-error)
+    - [Invalid training dataset](#invalid-training-data-set)
+    - [Invalid SAS URL](#invalid-sas-url)
+  - [Generic Error](#generic-error)
+- [Unexpected time to build a custom model](#unexpected-time-to-build-a-custom-model)
+- [Form Recognizer errors](#form-recognizer-errors)
+- [Logging](#logging)
+
+## Troubleshooting Errors
+
+### Handling RestError
+
+Form Recognizer service methods throw [`RestError`] on failure.
+
+The RestError raised by the Azure Form Recognizer client library includes detailed error response information that provides useful insights into what went wrong and includes corrective actions to fix common issues.
+This error information can be found in the `RestError#message` property.
+
+### Build model error
+
+Build model errors usually occur when trying to build a custom model. The most common scenarios when this might occur is, if you are building the model with an
+[Invalid data set](#invalid-training-data-set) or an [Invalid SAS Url](#invalid-sas-url).
+
+#### Invalid training data set
+
+This error indicates that the provided data set does not match the training data requirements.
+Learn more about building a training data set, [here](https://aka.ms/customModelV3).
```

Comment:
```suggestion
Learn more about building a training data set [on Microsoft Learn by following this link](https://aka.ms/customModelV3).
```

Avoid using links as opaque nouns ("here", "link", etc.) and offer some explanation of where the link will take you.